src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -129,6 +133,46 @@ public function panelAction(Request $request, $token)
     }
 
     /**
+     * Replays an action.
+     *
+     * @param Request $request The current HTTP Request
+     * @param string  $token   The profiler token
+     *
+     * @return RedirectResponse A RedirectResponse instance
+     */
+    public function replayAction(Request $request, $token)
+    {
+        $profile = $this->profiler->loadProfile($token);
+
+        /** @var ConfigDataCollector $configCollector */
+        $configCollector = $profile->getCollector('config');
+        $replayer = new Client($kernel = new \AppKernel($configCollector->getEnv(), $configCollector->isDebug()));
```

Comment:
You cannot assume the `AppKernel` class always exists and has this constructor signature... The kernel class can be renamed, so it must not be hardcoded here.